### PR TITLE
Enh: Improve smart reply UI

### PR DESCRIPTION
**Pull request by hamza221**
| B | A |
|--------|--------|
| image | image |

### CODE REVIEW

File: lib/Service/AiIntegrations/AiIntegrationsService.php [hunk 1]
```diff
@@ -178,7 +178,7 @@ public function getSmartReply(Account $account, Mailbox $mailbox, Message $messa
 			throw new ServiceException('Text processing is not available in your current Nextcloud version', 0, $e);
 		}
 		if (in_array(FreePromptTaskType::class, $manager->getAvailableTaskTypes(), true)) {
-			$cachedReplies = $this->cache->getValue('smartReplies_'.$message->getId());
+			$cachedReplies = $this->cache->getValue('smartReplies_'.$message->getUid());
```
Comment (ChristophWurst) [inline lib/Service/AiIntegrations/AiIntegrationsService.php:181]: how is this related to the UI enhancements? UIDs are not globally unique. Wouldn't you have a conflict for two identical UIDs in two mailboxes?

Comment (hamza221) [reply lib/Service/AiIntegrations/AiIntegrationsService.php:181]: Git add . 🥲
Was not supposed to be in the pr but I noticed that caching was broken and forgot to discard it.
